app/tests/tests.c

```diff
@@ -22,4 +22,7 @@ STATIC_COMMAND("bench", "miscellaneous benchmarks", &benchmarks)
 STATIC_COMMAND("fibo", "threaded fibonacci", &fibo)
 STATIC_COMMAND("spinner", "create a spinning thread", &spinner)
 STATIC_COMMAND("cbuf_tests", "test lib/cbuf", &cbuf_tests)
+#ifdef WITH_LIB_FS
```

Comment:
here and elsewhere, can you replace this with #if instead of #ifdef? There's a convention in the code for using it that way.